control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1623,8 +1623,9 @@ class P4RuntimeEntriesConverter {
             // earlier.
             // For e.g. 16 bit key has a max value of 65535, Range of (1..65536)
             // will be converted to (1..0) and will fail below check.
-            if (*start > *end)
-                ::error("%s Invalid range for table entry", kr->srcInfo);
+	    if (*start > *end)
```

Comment:
He said that he didn't like the error message produced.

Reply:
I restored the original code and checked in.